### ISSUE TITLE
Seek bar time overflow after few hours

### ISSUE THREAD

**Issue by micha1pu** [xbmc/xbmc#14564]
### Describe the bug
I’m having problem with master build of CoreElec, but also same problem happened on LibreElec 8.2 on S905 venz v10 box. After few hours playing playlist with videos freeze at the end of the video and doesn’t go to the next video, but seek bar is showing and counting time, but much longer than actual video length.

## Expected Behavior
Video player will play next video in playlist

## Actual Behavior
There is only black screen with overflow timeline, but timeline is still counting

### To Reproduce

1. plug in USB flash with multiple video in root directory

2. script in /storage/.kodi/userdata/autoexec.py 

> import xbmc
> import os
> 
> videoPath = os.listdir("/media/")
> if not videoPath:
>   exit()
> path = '/media/' + videoPath[0] + '/'
> cmd = 'xbmc.PlayMedia(' + path +',isdir)'
> 
> xbmc.executebuiltin(cmd)
> xbmc.executebuiltin('xbmc.PlayerControl(repeatall)')
> xbmc.executebuiltin("Action(Fullscreen)")

3. turn on device

### Debuglog

The debuglog can be found here:
[debug log](https://pastebin.com/raw/huEq6GRg)

### Screenshots 
Here are some links or screenshots to help explain the problem:

![freeze](https://user-images.githubusercontent.com/42468971/46719516-63f0e280-cc6e-11e8-85a6-02233d44be6d.jpg)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [x ] Linux
 - [ ] OSX
 - [ ] Raspberri-Pi
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
CoreELEC (community): devel-1538930643 (LePotato.arm)

 - Kodi version:
18.0-BETA3 Git:18.0b3-Leia
have same issue with 17.* versions

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
Sorry, but if you are using CoreELEC, please head over to their forums as it might probably be CE specific. 

If you are able to reproduce your issue on Ubuntu or probably Windows, please feel free to open another report. And then please make sure you provide a full debug log. We always need the full log and not only snippets from it.

Closed

**Comment by micha1pu** [comment]
Thank you for info. Log is 650Mb, so it was too big to upload, but i can provide later. I posted it on CoreElec forum first, but I got reply it is happening on windows too, so maybe kodi problem and they headed me here... I had same issue on LibreElec too. Anyway, i will try ubuntu version and I will post it here if there will be any problem.

**Comment by VelocityRa** [comment]
@micha1pu If you zip the log it will get much smaller.

**Comment by dguyton** [comment]
Adding a bit of anecdotal info FWIW. I have observed similar behavior in remote browser view of Kodi (over port 8080) on a LAN, with stock Kodi v18.1 running on headless Ubuntu 16.04.5 LTS server. At end of movies, if the movie was paused and resumed at any point during playback, the timer continues after end of total time of the movie, and continues indefinitely. 

I realize that's not especially helpful w/out more details, but I have observed this from two diff stock builds on the same platform described above. Perhaps that info will help someone better isolate the issue. It's not isolated to CoreElec.
